src/index.html

```diff
@@ -1,17 +1,179 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
-</html>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <!-- <div class="form-field"> -->
+          <label for="Surname">
+            Surname:
+            <input type="text" 
+            name="lastname" 
+            autocomplete="off" 
+            id="Surname">
+          </label>
+        <!-- </div> -->
+
+        <div class="form-field">
+          <label for="Name">
+            Name:
+            <input type="text" 
+            name="firstname" 
+            id="Name"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="are-You">
+            How old are You?
+            <input type="number" 
+            name="years" 
+            id="are-You"
+            value="12"
+            min="0"
+            max="100">

```

Comment:
```suggestion
            <input 
              type="number" 
              name="years" 
              id="are-You"
              value="12"
              min="0"
              max="100"
            >
```
Закрывающая скобка должна быть на одной линии с открывающей